index.js

```diff
@@ -15,7 +16,8 @@ class Inspector {
       }
     }
 
-    this.filename = filename || require?.main?.filename
+    const pearFilename = global.Pear && path.join(global.Pear.config?.dir, global.Pear.config?.main)
+    this.filename = filename || pearFilename || require?.main?.filename || process.argv[1]
```

Comment:
I think it will crash when reaching the last option (`process.argv[1]`), because process isn't imported.

For which flow would that occur? Because we could also just throw if this.filename still isn't defined after the first 3 options